automation_tools/__init__.py

```diff
@@ -1126,6 +1129,8 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     :param str certificate_url: where to fetch a fake certificate.
 
     """
+    # Fetch the Satellite Version information.
+    satellite_version = os.environ.get('SATELLITE_VERSION')
```

Comment:
Better to fail if the satellite_version is not defined and before doing any change on the server